src/parcel.js

```diff
@@ -26,6 +26,7 @@ export default class Parcel extends React.Component {
     }
   }
   componentDidMount() {
+    this.unmounted = false;
```

Comment:
I ran this code locally to see which tests failed in strict mode when this line was commented versus uncommented.

All tests pass regardless of whether this line of code is included. Please add a new test that demonstrates that `this.unmounted = false` is necessary.

Reply:
cc @nickbosland thanks

Reply:
Thanks for the CC. I somehow missed the the notification. 

I have updated the `@testing-library/react` to [14.2.0](https://github.com/testing-library/react-testing-library/releases/tag/v14.2.0) which actually adds the option to configure whether strict mode is used in the tests or not. I also messed up the property name of the option somehow 🤦. I had to move the `configure` to the `beforeEach` as it does not work in the `beforeAll`.

Now multiple tests fail when this is line is removed.

Reply:
Thanks @nickbosland — cc @joeldenning would you be happy to proceed with the PR now? Appreciate you taking a look!

Reply:
@MilanKovacic @joeldenning any chance this can be reviewed again ?

Reply:
I've contacted Joel to re-review it.